src/BTConnectionManager.cpp

```diff
@@ -67,6 +79,9 @@ class BTConnectionManager::Private {
             }
         });
     }
+
+    QBluetoothLocalDevice* localDevice;
```

Comment:
As with the other bits in this section, please initialise these here, rather than in the ctor, to match the style of the rest of this class, thanks :)